tensorflow_datasets/audio/commonvoice.py

```diff
@@ -138,18 +140,16 @@ def _split_generators(self, dl_manager):
 
   def _generate_examples(self, audio_path, label_path):
     """Generate Voice samples and statements given the data paths.
-
```

Comment:
Why removing new lines here ?

Reply:
i'am sorry it's seem i accidentally remove the space, i will revert it. thank you for noticing.